disnake/webhook/sync.py

```diff
@@ -439,10 +439,14 @@ def edit(
             Files will be appended to the message, see the ``attachments`` parameter
             to remove/replace existing files.
         attachments: List[:class:`Attachment`]
-            A list of attachments to keep in the message. If ``[]`` is passed
-            then all existing attachments are removed.
+            A list of attachments to keep in the message.
+            If ``[]`` or ``None`` is passed then all existing attachments are removed.
```

Comment:
parameter annotation not updated

Reply:
https://github.com/DisnakeDev/disnake/pull/457/commits/3dc30b1af1fdbfd9fddbd3ab07370b36cae42029